### ISSUE TITLE
Terminal-like collapsible text block

### ISSUE THREAD

**Issue by admnd** [BookStackApp/BookStack#5311]
### Describe the feature you'd like

Add a new kind collapsible text block that looks like a Linux terminal with customisable prompts / colors (see attached images).

![term_root](https://github.com/user-attachments/assets/2a00a1ce-6acc-4bcf-93eb-b515792e166f)
![term_user](https://github.com/user-attachments/assets/ba0e2c5c-5aac-4487-804a-03c9d99f1fc2)

### Describe the benefits this would bring to existing BookStack users

Make computer-related wiki articles nicer and more readable.

### Can the goal of this request already be achieved via other means?

No idea. Perhaps some kind of template? Found nothing on the internets so far.

### Have you searched for an existing open/closed issue?

- [X] I have searched for existing issues and none cover my fundamental request

### How long have you been using BookStack?

Under 3 months

**Comment by Hallsie** [comment]
Is the code block not enough inside the collapsible window?  I'm guessing you are asking that the whole thing would be a collapsible code block that you add a title for and it's a code block inside that only?

**Comment by Hallsie** [comment]
> Is the code block not enough inside the collapsible window? I'm guessing you are asking that the whole thing would be a collapsible code block that you add a title for and it's a code block inside that only?

Can you put a code block inside a collapsible window?  when I press save it pulls it back out into my main page and will not go into the collapsible block.

**Comment by PatrickGlatz** [comment]
Yeah, the codeblocks are only very basic and lack of headline, description which language and the ability to react on different keywords properly. There are different feature request in this regards

**Comment by admnd** [comment]
Having a feaure like this one will greatly improve readability :  You can mix "real" code with terminal commands without confusion about which is which.  Or perhaps a template could do the job without duplicating functionalities?

**Comment by ssddanbrown** [comment]
BookStack is now [managed on Codeberg](https://codeberg.org/bookstack/bookstack).

As part of this change, we're closing all existing GitHub issues and pull requests.
You'll instead find this issue on Codeberg here: 
https://codeberg.org/bookstack/bookstack/issues/5311
  
Any further discussion for this will now be done on Codeberg.
Therefore this GitHub thread will be locked.
